extension/package.json

```diff
@@ -6,6 +6,7 @@
     "lint": "./node_modules/.bin/eslint --ignore-path ../.gitignore --ext .js --ext .jsx .",
     "test": "./node_modules/.bin/_mocha --use_strict --recursive ./test/unit",
     "cover": "./node_modules/.bin/nyc npm test && ./node_modules/.bin/nyc report --reporter=text-lcov | coveralls",
+    "newman": "./node_modules/.bin/newman run ./test/newman/collection.json --globals ./test/newman/globals.json --environment ./test/newman/environment.json",
```

Comment:
since this assumes an environment file has to exist, you will need to create an automated environment file creator. Please see other plugins on how to do it, specifically the `newman-init` script. I believe it is in magento-user extension or one of the shopify ones.

Reply:
we will add this on Rainers PR